scripts/notarize-macos-dmg.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash` would be more robust (but I keep forgetting this for my own scripts 😅)

Reply:
Although, on macOS we can assume `/bin/bash`...